Lets Do This/Views/Campaign/LDTCampaignDetailCampaignCell.m

```diff
@@ -41,19 +41,6 @@ - (void)awakeFromNib {
     self.staticInstructionLabel.text = @"When you’re done, submit a pic of yourself in action. #picsoritdidnthappen";
 }
 
-- (UICollectionViewLayoutAttributes *)preferredLayoutAttributesFittingAttributes:(UICollectionViewLayoutAttributes *)layoutAttributes {
-    UICollectionViewLayoutAttributes *attributes = [[super preferredLayoutAttributesFittingAttributes:layoutAttributes] copy];
-
-    [self setNeedsLayout];
-    [self layoutIfNeeded];
```

Comment:
The [honghaoz demo states](https://github.com/honghaoz/Dynamic-Collection-View-Cell-With-Auto-Layout-Demo#uicollectionviewcell):
> You may also need a configure function, make sure in this function, call self.setNeedsLayout() and self.layoutIfNeeded()

We're calling `setNeedsLayout` and `layoutIfNeeded` on the cell when we configure the sizing cell here: https://github.com/DoSomething/LetsDoThis-iOS/blob/5a598291492a58ac832cb344b31ca8dec61cf66e/Lets%20Do%20This/Controllers/Campaign/LDTCampaignDetailViewController.m#L360

The demo seems to suggest that 
> In cellForItemAtIndexPath:, cell also need configured and layout its subviews

This branch as is just calls it on the sizing cell, not the cell to display within `cellForItemAtIdexPath`.